internal/backends/linuxkms/calloop_backend.rs

```diff
@@ -240,9 +238,16 @@ impl i_slint_core::platform::Platform for Backend {
         while !quit_loop.load(std::sync::atomic::Ordering::Acquire) {
             i_slint_core::platform::update_timers_and_animations();
 
-            adapter.render_if_needed(mouse_position_property.as_ref())?;
+            if let Some(adapter) = self.window.borrow().as_ref() {
```

Comment:
Style: maybe return the has_active_animation from this if: `let has_active_animaiton = if let ... `

Reply:
Good idea :)